### PR TITLE
Mention -q and -s in the docs

### LINKED ISSUE #11507: Documentation: "Managing pytest’s output" should mention `-q` and `-s`
Looking at https://docs.pytest.org/en/7.1.x/how-to/output.html, I am missing references to command line options `-q` (`--quiet `) and `-s` (`--capture`), which clearly have an influence on `pytest`s output.

### PR DESCRIPTION
**Pull request by ziadk**
- Closes #11507.
- Added references to command line options `-q (--quiet )` ,`--capture` and `-s (--capture)` in the output documentation.

### CODE REVIEW

**Comment by ziadk** [comment]
@nicoddemus could you please review my request, relating to #11507 ?

File: doc/en/how-to/output.rst [hunk 1]
```diff
@@ -15,6 +15,15 @@ Examples for modifying traceback printing:
     pytest --showlocals     # show local variables in tracebacks
     pytest -l               # show local variables (shortcut)
     pytest --no-showlocals  # hide local variables (if addopts enables them)
+    pytest --quiet          # quiet - less verbose - mode
```
Comment (nicoddemus) [inline doc/en/how-to/output.rst:18]: Those look nice, but I think they should appear in the `Verbosity` section instead: although they do affect tracebacks, seems like `Verbosity` is a better place to mention them.

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Please see comments.

File: doc/en/how-to/output.rst (hunk 1 above)
Comment (ziadk) [reply doc/en/how-to/output.rst:18]: Will do. Would you like me to move all the lines that I have added to `Verbosity `?

File: doc/en/how-to/output.rst [hunk 2]
```diff
@@ -36,6 +42,13 @@ option you make sure a trace is shown.
...
 --------------------------------------------------
 
+Examples for modifying printing verbosity:
+
+.. code-block:: bash
+
+    pytest --quiet          # quiet - less verbose - mode
+    pytest -q               # quiet - less verbose - mode (shortcut)
+
 The ``-v`` flag controls the verbosity of pytest output in various aspects: test session progress, assertion
```
Comment (nicoddemus) [inline doc/en/how-to/output.rst:52]: It is a bit strange to list `-q` and `--quiet`, and right in the following paragraph discuss `-v`, which is not shown as example.

Please include `-v`, `-vv` and `-vvv` in the list above here as well.

Comment (ziadk) [reply doc/en/how-to/output.rst:52]: Will do. I'm sorry I just try to not add anything I wasn't told

**Comment by ziadk** [comment]
@nicoddemus. I am sorry to bother, but I have no idea why the checks keep failing. Could you maybe help me ?

**Comment by ziadk** [comment]
@nicoddemus Should be all good now.

File: changelog/11507.doc.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Added documentation lines for command line options -q (--quiet ).
```
Comment (nicoddemus) [inline changelog/11507.doc.rst:1]: Probably not needed. 👍

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**